### ISSUE TITLE
Found a reason for WinForms Designer to 'hang' when loading

### ISSUE THREAD

**Issue by simmotech** [dotnet/winforms#6245]
Symptom:
Opening a Form or User Control shows "Loading designer..." and never completes.

My main applications both did this for months and I assumed it was because I was using pre-release versions of VS 2022.
But the same happened with the final release so today I got around to researching what the problem was.

I turned on Verbose output for the WinForms Designer and spotted an exception about a missing Type in a resource file.
Although the log showed many more entries after this (and the apps run fine even with the missing type), the designer never loads but doesn't show any error message to give any clue as to why.

I tried edited the resources.rex file and just manually corrected the version numbers:
`<value>..\Resources\SmartLibrarianLogo.svg;DevExpress.Utils.Svg.SvgImage, DevExpress.Data.v21.1, Version=21.1.5.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a</value>
`
to
`<value>..\Resources\SmartLibrarianLogo.svg;DevExpress.Utils.Svg.SvgImage, DevExpress.Data.v21.2, Version=21.2.3.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a</value>
`

After restarting VS, there are no errors in the log and the designer works!

I'm not clear why the designer loading process log seems to continue when there is a fatal problem. It should either be able to skip over this issue or show a dialog box to say there is a problem.

**Comment by RussKie** [comment]
I appreciate you have spent the time to raise the issue, and shared a fix that you have found. However issues with VS and VS Designer should be sent via VS Feedback channel (the icon in the top right hand side corner). The reason for this - the VS Feedback tool captures a lot of additional information (such as screenshots, memory dumps, stack traces, assembly versions, etc.) that can help us to investigate your issue and identify the cause.

Please post the link to the VS Feedback issue here, so that our team can track it.

**Comment by simmotech** [comment]
The issue _was_ identified - no additional information needed.
I feel I have done my bit in reporting not only a problem but an easy way for you to reproduce it.
So to have the issue ignored and closed immediately rankles a bit.

I have logged the issue via Visual Studio as you requested but I have moved past this now - I won't be breaking my code again to send memory dumps/stack traces etc. so if their bot closes it after 14 days after asking for more info....
